### ISSUE TITLE
Inconsistent use of PAN_MAX in the pan law (in NotePropertiesRuler )

### ISSUE THREAD

**Issue by oddtime** [hydrogen-music/hydrogen#1167]
I have noticed one inconsistent use of PAN_MAX in src/gui/src/PatternEditor/NotePropertiesRuler.cpp line 487:
```
	if ( fValue > PAN_MAX ) {
		fPanL = 2*PAN_MAX - fValue;
		fPanR = PAN_MAX;
	} else {
		fPanL = PAN_MAX;
		fPanR = fValue;
	}
```

I guess it is an oversight , since there PAN_MAX = 0.5 = middle point in pan parameter domain for the widget ( i.e. [0;1] ). 
The computation is correct but PAN_MAX refers to the gain weights and not to the pan parameter, in addition if we want to use PAN_MAX explicitly in the pan law, the expressions should be different

I have modified it in pull req #1015 which is still open.

**Comment by cme** [comment]
Excellent. I think this is my fault. All the more argument for commonising code like this! :)

**Comment by oddtime** [comment]
> All the more argument for commonising code like this!

We could actually, we already have the pan law as static functions in `Sampler`

**Comment by oddtime** [comment]
Actually this part could get even simpler... #1242

**Comment by theGreatWhiteShark** [comment]
Fixed in #1273
